modules/installation-extracting-infraid.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
```

Comment:
Shouldn't this file be called ...-clusterid or ...-clustername?

Reply:
It's called "infraID" when you extract it from the metadata and "InfrastructureName" in the AWS templates.